content/content-tool/tool/src/java/org/sakaiproject/content/tool/FilePickerAction.java

```diff
@@ -580,7 +580,43 @@ protected String buildSelectAttachmentContext(VelocityPortlet portlet, Context c
 		context.put("stlang",srb);
 
 		ToolSession toolSession = sessionManager.getCurrentToolSession();
-		context.put("maxUploadFileSize", toolSession.getAttribute(STATE_FILE_UPLOAD_MAX_SIZE));
+
+		String uploadCeiling = ServerConfigurationService.getString("content.upload.ceiling");
+		String uploadMax = (String) toolSession.getAttribute(STATE_FILE_UPLOAD_MAX_SIZE);
+
+		if (!uploadCeiling.isBlank()) 
+		{
+			int uploadCeilingNum = Integer.MAX_VALUE;
+			int uploadMaxNum = Integer.MAX_VALUE;
+			try
+			{
+				uploadCeilingNum = Integer.parseInt(uploadCeiling);
+			}
+			catch(Exception e)
+			{
+			}
```

Comment:
lets add a log message to these ignored exceptions, they can be debug messages if they should be ignored

Reply:
maybe NumberUtils would be useful here?

Reply:
@ern Thank you, I have updated the catch blocks in this new pull request: [12666](https://github.com/sakaiproject/sakai/pull/12666)